doc/source/user_guide/utilities.rst

```diff
@@ -81,17 +81,27 @@ The default operational mode is ``inplace=False`` (a new signal object is return
 Utilities for spectral maps
 ===========================
 
-The function :py:meth:`~.signals.common_luminescence.CommonLumi.crop_edges`
-removes the specified number of pixels from all four edges of a spectral map.
+The function :py:meth:`~.utils.axes.crop_edges`
+removes the specified number of pixels or % from the four edges of a spectral map, from the edges inwards. It takes a list of `Signals` and cropping can happen uniformly on all sides or by specifying the cropping range for each axis or each side. If the navigation axes shape across the list of signals is different, all signals can be rebinned to match the shape of the first signal in the list.
 It is a convenience wrapper for the ``inav`` `method in HyperSpy
 <https://hyperspy.org/hyperspy-doc/current/user_guide/signal.html#indexing>`_.
 
 .. code-block:: python
 
-    >>> s.crop_edges(crop_px=2)
+    >>> signals = [cl_map, sem_image]
+    >>> signals
+    [CLSpectrum <256,256|1024>, Signal2D <128,128|1>]
+    >>> signals_cropped = lum.utils.axes.crop_edges(crop_range=5, crop_units="%", rebin_nav=True)
```

Comment:
```suggestion
    >>> signals_cropped = lum.utils.axes.crop_edges(signals, crop_range=5, crop_units="%", rebin_nav=True)
```

As this is acting on signals and not axes, would it make more sense to have this function define in `lum.utils.crop_edges`?

Reply:
Yes, also makes it less complicated to call.

Reply:
Implemented. Had to shift some of the documentation accordingly. Also, following the same reasoning, I moved the `join_spectra` function to utils too, as it acts on signals and not axes.